apph-back/src/test/java/com/viseo/apph/PhotoTest.java

```diff
@@ -283,38 +248,14 @@ public void testDownloadUserNotAllowed() {
     public void testFailUserNotFind() {
         //GIVEN
         createPhotoController();
-        String jws = Jwts.builder().claim("login", "dumb_toto").setExpiration(new Date(System.currentTimeMillis() + 20000)).signWith(JwtConfig.getKey()).compact();
+        User user = new User().setLogin("dumb_toto");
+        when(utils.getUser()).thenReturn(user);
         //WHEN
-        ResponseEntity responseEntity = photoController.getUserPhotos(jws, 1, 1);
+        ResponseEntity responseEntity = photoController.getUserPhotos(1, 1);
         //THEN
         Assert.assertEquals(responseEntity.getStatusCode(), HttpStatus.FORBIDDEN);
     }
```

Comment:
Ce test n'est pas censé fail ? En tout cas j'ai pas l'impression que le test va fail à cause d'un user not find

Reply:
Je retire ce test car l'user existe forcement si il a réussi à acceder au controlleur sinon le filtre aurait rejette la requête

Reply:
Justement, tu ne l'as pas retiré le test, elle est encore présent.

Reply:
Oui je retire dans le sens ou je vais le retirer